ynr/settings/testing.py

```diff
@@ -40,3 +40,23 @@ def __getitem__(self, item):
 MEDIA_ROOT = mkdtemp()
 
 FRONT_PAGE_CTA = "BY_ELECTIONS"
+
+
+# override these settings to safe values if they are set from the env
+TWITTER_APP_ONLY_BEARER_TOKEN = None
```

Comment:
I can't leave an inline comment on this in base.py because I didn't modify the line, so I'll bung it here..

This variable is not set in prod. I think we said we don't need it any more.
However, we have a load of code referencing this variable so I left it alone in `base.py`.
https://github.com/search?q=repo%3ADemocracyClub%2Fyournextrepresentative%20TWITTER_APP_ONLY_BEARER_TOKEN&type=code
Should all the code that uses this var just be deleted?